cmd/util/ledger/migrations/accounts.go

```diff
@@ -137,6 +132,73 @@ func appendKeyForAccount(accounts *state.Accounts, addressInHex string, encodedK
 	return nil
 }
 
+type view struct {
+	Parent *view
+	Ledger *led
+}
+
+func newView(payloads []ledger.Payload) *view {
+	return &view{
+		Ledger: newLed(payloads),
+	}
+}
+
+func (v *view) NewChild() state.View {
+	payload := make([]ledger.Payload, 0)
+	ch := newView(payload)
+	ch.Parent = v
+	return ch
+}
+
+func (v *view) DropDelta() {
+	v.Ledger.payloads = make(map[string]ledger.Payload)
+}
+
+func (v *view) MergeView(o state.View) error {
+	var other *view
+	var ok bool
+	if other, ok = o.(*view); !ok {
+		return fmt.Errorf("can not merge: view type mismatch (given: %T, expected:Delta.View)", o)
```

Comment:
```suggestion
		return fmt.Errorf("view type mismatch (given: %T, expected:Delta.View)", o)
```
pretending "cannot merge" to the error is better to be done by the caller.